.travis.yml

```diff
@@ -0,0 +1,15 @@
+decorator==4.0.11
+Django==1.10.5
+ipython==5.1.0
+ipython-genutils==0.1.0
+pexpect==4.2.1
+pickleshare==0.7.4
+pkg-resources==0.0.0
+prompt-toolkit==1.0.9
+psycopg2==2.6.2
+ptyprocess==0.5.1
+Pygments==2.1.3
+simplegeneric==0.8.1
+six==1.10.0
+traitlets==4.3.1
+wcwidth==0.1.7
```

Comment:
You seem to have mis-named your `travis.yml` file. This is a `requirements` file.